config.json

```diff
@@ -50,6 +50,10 @@
     },
     {
       "name": "12.x",
+      "branch": "branch/v12"
```

Comment:
@alexfornuto I think you need to move `latest: true` here, right?

Reply:
Yup, and add Deprecated to v9. Thanks!